templates/repo/actions/workflow_dispatch.tmpl

```diff
@@ -0,0 +1,72 @@
+<div class="ui ignored info message project-column-header" >
```

Comment:
```suggestion
<div class="ui ignored info message">
```

This class is related to Projects feature, I think it can likely be safely removed, but please try it first.